ovc/machines.go

```diff
@@ -302,7 +302,7 @@ func (s *MachineServiceOp) Template(machineID int, templateName string) error {
 	machineMap["machineId"] = machineID
 	machineMap["templateName"] = templateName
 
-	_, err := s.client.Post("/cloudapi/machines/createTemplate", machineMap)
+	_, err := s.client.Post("/cloudapi/machines/createTemplate", machineMap, OperationalActionTimeout)
```

Comment:
consider making this a dataopertion createtemplate does a full disk copy from one ovs volume to another